tests/testthat/test-are-all-close.R

```diff
@@ -0,0 +1,51 @@
+test_that("are_all_close_correct_true", {
+
+  # 1) correctly returns TRUE
```

Comment:
-1

Don't use comments for describing what is being done, since that should usually be evident by reading the code and they just add clutter. Use comments to describe why something is done.

And in this case the name of the test on line one can contain all this info